Source/WebCore/platform/mediastream/gstreamer/GStreamerCapturer.cpp

```diff
@@ -170,6 +170,7 @@ void GStreamerCapturer::setupPipeline()
     m_sink = makeElement("appsink");
 
     gst_app_sink_set_emit_signals(GST_APP_SINK(m_sink.get()), TRUE);
+    g_object_set(GST_BASE_SINK(m_sink.get()), "enable-last-sample", FALSE, nullptr);
```

Comment:
niptick: There's no need to cast m_sink for g_object_set, such as happens in the line below.